languages/javascript/exercises/concept/array-analysis/.docs/introduction.md

```diff
@@ -0,0 +1,23 @@
+In JavaScript, an array is a list-like structure with no fixed length which can hold any type of primitives or objects, or even mixed types. The array elements can be accessed by their index. Arrays are also given a bunch of built-in methods.
+
+In this exercise we will use built-in methods to analyse the contents of an array, instead of using a `for` loop, or `.forEach`.
```

Comment:
This comment refers to the exercise itself, which means that it should be part of the `instructions.md` file instead of the `introduction.md` file.

Reply:
which part should i move ? is it starting from "In this exercise we will ...." or the whole instruction.md ?

Reply:
> In this exercise we will use built-in methods to analyse the contents of an array, instead of using a `for` loop, or `.forEach`.